vnext/Microsoft.ReactNative/Views/VirtualTextViewManager.cpp

```diff
@@ -69,6 +71,10 @@ bool VirtualTextViewManager::UpdateProperty(
   } else if (propertyName == "textTransform") {
     auto node = static_cast<VirtualTextShadowNode *>(nodeToUpdate);
     node->transformableText.textTransform = TransformableText::GetTextTransform(propertyValue);
+  } else if (propertyName == "backgroundColor") {
+    static_cast<VirtualTextShadowNode *>(nodeToUpdate)->m_highlightData->isHighlighted =
```

Comment:
why do we need the 2 properties? can we use color (or optional<color>) as the indicator?

Reply:
You're right, I might as well be comparing with a null winrt::Color here. I'll change this.

Reply:
Changed to std::optional